README.md

```diff
@@ -106,6 +106,13 @@ require("neotest-playwright").adapter({
     -- filter_dir = function(name, rel_path, root)
     --   return name ~= "node_modules"
     -- end,
+
+    -- Custom criteria for a file path to be a test file. Useful in large projects or projects with peculiar tests folder structure.
+    -- is_test_file = function(file_path)
```

Comment:
Does this imply that we also need to manually specify the path for filtering tests? Considering the already substantial amount of manual effort required for switching (or adding new) projects?

Reply:
If I understand the question correctly, yes. 

In monorepos consisted of many subprojects with tests we would require such extra effort since more than 1 adapters might register some tests as their own if the filtering is only based on file extension matching.

Let me know if i answered your question.

Reply:
This is optional, most users will probably not need to declare a custom `is_test_file` function. It can, however, come in handy in certain [situations](https://github.com/thenbe/neotest-playwright/issues/24#issuecomment-1951423647).

Reply:
Thanks for the explanations! At least it's only needed in certain situations like resolving adapter collisions, that makes it less laborious than i thought. I still wonder if it's possible to automatically discover the tests somehow?

Maybe by parsing the `playwright.config.ts` file associated with the registered test adapter/project and read properties like `testMatch` or `testDir`? I believe the vscode extension accomplishes something similar.

When I'm working on projects with both Vitest unit tests and Playwright component tests, I usually have to specify `defineConfig({ testMatch: '**/?(*.pw.)+(spec|test).ts })` for Playwright, and/or `defineConfig({ test: { exclude: ['**/*.unit.{test,spec}.?(c|m)[jt]s?(x)'] } })` for Vitest _(AKA what is proposed to use in `is_test_file`)_. Can't this be reuse this somehow?

Reply:
Reply here: https://github.com/thenbe/neotest-playwright/issues/24#issuecomment-1953436311